subprojects/docs/src/docs/userguide/running-builds/init_scripts.adoc

```diff
@@ -40,9 +40,9 @@ One main limitation of init scripts is that they cannot access classes in the `b
 There are several ways to use an init script:
 
 * Specify a file on the command line. The command line option is `-I` or `--init-script` followed by the path to the script. The command line option can appear more than once, each time adding another init script. The build will fail if any of the files specified on the command line does not exist.
-* Put a file called `init.gradle` (or `init.gradle.kts` for Kotlin) in the `__USER_HOME__/.gradle/` directory.
```

Comment:
It think we need to define what `USER_HOME` is. There's nothing as a `USER_HOME` environment variable.
On unices it is `$HOME`, on Windows I don't know.

Same for `GRADLE_HOME`.

Defining these upfront would be a good place to add a link to the installation documentation.